libbpf-rs/src/tc.rs

```diff
@@ -0,0 +1,288 @@
+use nix::errno;
+use nix::errno::Errno::{EEXIST, EINVAL, ENOENT};
+
+use crate::*;
+
+pub type TcAttachPoint = libbpf_sys::bpf_tc_attach_point;
+pub const INGRESS: TcAttachPoint = libbpf_sys::BPF_TC_INGRESS;
+pub const EGRESS: TcAttachPoint = libbpf_sys::BPF_TC_EGRESS;
+pub const CUSTOM: TcAttachPoint = libbpf_sys::BPF_TC_CUSTOM;
```

Comment:
I think we should rename it with the preface TC_ (or maybe not even redeclare it here and just use libbpf_sys's declaration everywhere?). I think ingress/egress/custom is too common and doesn't pertain to just tc (eg cgroup uses it too afaik).